web_export_view/static/src/js/web_export_view.js

```diff
@@ -85,7 +85,8 @@ openerp.web_export_view = function (instance) {
                                 }
                             }
                             else if (cell.classList.contains("oe_list_field_integer")) {
-                                export_row.push(parseInt(text));
+                                tmp2 = text.replace(instance.web._t.database.parameters.thousands_sep, "");
```

Comment:
I think you need to use the __var__ keyword to scope the variable tmp2 properly. Alternatively, don't use a temporary variable but perform the operation inline instead.